src/pages/community/[slug].astro

```diff
@@ -0,0 +1,17 @@
+---
+import Layout from "../../layouts/layout.astro";
+import Container from "../../components/page-container.astro";
+import directus from "../../lib/directus";
+import { readItems } from "@directus/sdk";
+
+export async function getStaticPaths() {
+  const pages = await directus.request(readItems("groups"));
+  return pages.map((page) => ({
+    params: { slug: page.slug },
+    props: page,
+  }));
+}
+const page = Astro.props;
+
+return Astro.redirect(page.link);
```

Comment:
This may not work, using Astro.redirect.
The reason is, because it's an SSG site. Worth a try though. If not, will require some JS probably

Reply:
It seems it works when I try it